openedx/core/lib/logsettings.py

```diff
@@ -12,14 +12,8 @@
 
 def get_logger_config(log_dir,
                       logging_env="no_env",
-                      tracking_filename="tracking.log",
```

Comment:
There's still a usage of this function in `lms/envs/static.py` that explicitly passes in the default values of `tracking_filename` and `debug`.

Reply:
I got rid of that usage as well: https://github.com/edx/edx-platform/pull/17374/files#diff-df66f685d5af5a8b29d5a6ec17ff0c8cL23